### ISSUE TITLE
Question - modules loader

### ISSUE THREAD

**Issue by SoundBot** [rjrudin/ml-javaclient-util#41]
I was debugging a long execution time of `mlLoadModules` and came across to this library. I didn't dive deep into the code, but it seems to me (and please correct me, if I'm wrong), by looking into `DefaultModulesLoader.java`, that certain things are sequential instead of parallel. Was there a good reason to do this?

**Comment by rjrudin** [comment]
@SoundBot I just released version 2.5.0 of ml-gradle, which includes 2.11.0 of ml-javaclient-util, where assets are now loaded in bulk instead of one-by-one. 

As for non-assets - options, services, transforms, namespaces - you make a good point, they could all be loaded in parallel once the assets have been loaded (services/transforms may have dependencies on assets). Actually - they may have dependencies on namespaces, I forget exactly how namespaces work (and I don't that feature is used much). But certainly loading options/services/transforms in parallel would speed things up, as each of those has to be loaded individually and takes a second or two to load. I'll close this and open a new ticket. Thanks!
